src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -22,7 +22,7 @@
 import com.google.codeu.data.Message;
 import com.google.gson.Gson;
 import java.io.IOException;
-import java.util.List;
```

Comment:
Why did you replace the import java.util.List to import java.util.*?

Reply:
I was trying something out with initializing my own list but I forgot to change it back because I ended up not going that route. I'll change it and commit again

Reply:
Ok sounds good!